doc/source/changes.rst

```diff
@@ -6,6 +6,13 @@ Changes/Hints for migration
 This chapter will give you hints on how to handle version migration, in particular regarding what you may need
 to adapt in your project configuration, database etc. when upgrading to a new version.
 
+Version 2.6.0
+-------------
+* New parameter 'expected_toc_length' allows to define a default table of content pages number avoiding a second
+call for the pdf extract in most cases. This value should be set if most of the PDF extracts have the same number
+of TOC pages.
+Default setting: 2
```

Comment:
That is omitted now.

A change in this pull request is that the pdf is built a second time whenever the numbers of toc pages disagree between extract_as_dict['nbTocPages'] (computed/expected/1, depending on the setting) and the one in the returned pdf.

Reply:
In the current version it is only rebuild if 'compute_toc_pages' is true.